src/bundle/Resources/public/scss/_global-search.scss

```diff
@@ -217,4 +217,14 @@
             }
         }
     }
+
+    &__shortcut {
+        display: inline-block;
+        line-height: calculateRem(21px);
+        font-size: $ibexa-text-font-size-medium;
+        padding: 0 calculateRem(4px);
+        background-color: $ibexa-color-light-300;
+        border-radius: calculateRem(4px);
+        font-weight: 400;
```

Comment:
Maybe we declare if we are using variables with font-weight or not. Right now we have $headings-font-weight: 700,
$ibexa-font-weight-bold: 600, $ibexa-font-weight-normal: normal;
@GrabowskiM @lucasOsti @tischsoic @dew326